data_loader.py

```diff
@@ -0,0 +1,73 @@
+import settings
+import json
+from googleapiclient.discovery import build
+from google.oauth2 import service_account
+from utils import _export_file
+from builder_client import BuilderClient
+from constants import PDF_PATH, SCOPES
+
+
+def load_file_data():
+    """
+    This script will bootstrap a guide in Builder with all files in given folder in Google Drive.
+    Be sure all of the settings in settings.py are up to date before running this.
+    """
+    builder_client = BuilderClient(settings.builder_api_key)
```

Comment:
nit: you could make this even more robust by creating util method that checks if the settings values have been updated and raise and error right away if they haven't been